spec/music_album_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../music_album'
+
+describe MusicAlbum do
+  describe '#initialize' do
+    before(:each) do
+      @album = MusicAlbum.new(true, Date.new(2012, 5, 31), false, 'Title')
+    end
+
+    it 'creates a new instance of MusicAlbum' do
+      expect(@album).to be_an_instance_of(MusicAlbum)
+    end
+
+    it 'should have attributes' do
+      expect(@album).to have_attributes(on_spotify: true, publish_date: Date.new(2012, 5, 31), archived: false,
+                                        title: 'Title')
+    end
+  end
```

Comment:
- [ ] Nice adding tests but you forgot to add tests for the `item` class as the requirements say  you should add tests for all the methods including the item's methods 
<img width="416" alt="Screenshot 2023-06-01 at 23 17 19" src="https://github.com/NicolasAndrehh/Things-Catalog/assets/61790587/a1790899-98b2-4b5b-9911-5c2f14018775">